packages/inventory-insights/src/Insights.js

```diff
@@ -92,46 +109,79 @@ class InventoryRuleList extends Component {
-                                    <SplitItem className="smallFontSizeOverride">
-                                        { value.resolution.has_playbook &&
-                                        <Fragment><CheckIcon className='successColorOverride'/> Ansible remediation </Fragment>
-                                        }
-                                    </SplitItem>
-                                </Split>
-                                <Section type='icon-group__with-major' className='batterySectionOverrides smallFontSizeOverride'>
-                                    <Battery label='Impact' severity={ rule.impact.impact }/>
-                                    <Battery label='Likelihood' severity={ rule.likelihood }/>
-                                    <Battery label='Total Risk' severity={ rule.total_risk }/>
-                                    <Battery label='Risk Of Change' severity={ value.resolution.resolution_risk.risk }/>
-                                </Section>
-                            </Fragment>
+                            title: <div key={ key }>
+                                { moment(rule.publish_date).fromNow() }
+                            </div>
+                        },
+                        {
+                            title: <div className='pf-m-center' key={ key } style={ { verticalAlign: 'top' } }>
+                                <Battery
+                                    label='Total risk'
+                                    labelHidden
+                                    severity={ rule.total_risk }
+                                />
+                            </div>
+                        }, {
+                            title: <div className='pf-m-center' key={ key } style={ { verticalAlign: 'top' } }>
+                                <Battery
+                                    label='Risk of change'
+                                    labelHidden
+                                    severity={ value.resolution.resolution_risk.risk }
+                                />
+                            </div>
+                        },
+                        {
+                            title: <div className='pf-m-center ' key={ key }>
+                                { value.resolution.has_playbook &&
+                                <CheckIcon className='successColorOverride'/> }
+                            </div>
                         }
                     ]
                 },
                 {
-                    parent,
+                    parent: key,
                     fullWidth: true,
                     cells: [{
                         title: <ReportDetails key={ `child-${key}` } report={ value } kbaDetail={ kbaDetail }/>
                     }]
                 }
             ];
+
+            const isValidSearchValue = searchValue.length === 0 || rule.description.search(new RegExp(searchValue, 'i')) !== -1;
```

Comment:
You don't really need to check `length === 0` right? Also can you split this into special function?

Reply:
yeah! need to check length === 0 here, when there's no text input we would like to show all results (rather than no results).  as for splitting it into a function, can do! wondering what yer looking to achieve by that?

Reply:
Oh okay, well with the function you don't have to recreate it for each row.